integration/mediation-tests/tests-mediator-1/src/test/resources/artifacts/ESB/synapseconfig/propertyMediatorConfig/dynamic_property_names.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<definitions xmlns="http://ws.apache.org/ns/synapse">
+    <proxy name="DynamicPropertiesGetPropertyEval" transports="https http" startOnLoad="true" trace="disable">
```

Comment:
We normally have separate files for different artifacts instead of one file definition in MI. Shall we divide this?

Reply:
Fixed